datafusion/sql/src/planner.rs

```diff
@@ -466,7 +467,14 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                     // Timestamp Without Time zone
                     None
                 };
-                Ok(DataType::Timestamp(TimeUnit::Nanosecond, tz.map(Into::into)))
+                let precision = match precision {
```

Comment:
Can I assume this is the standard between vector api and presion?

Reply:
I think the relationship between vector api and precision can be briefly described